packages/common/src/index.ts

```diff
@@ -79,17 +93,19 @@ export default class Common {
     })
   }
 
-  private static _getChainParams(chain: string | number): Chain {
+  private static _getChainParams(chain: string | number, customChains?: Chain[]): Chain {
```

Comment:
Here `customChains` is an argument - can we not use `this.customChains` and remove the argument?

Reply:
No, since the method is static and custom chains from the instance are not available yet. This is a private method though, so should be unproblematic.

Reply:
Ah - woops - did not see that this was a `static` method.